packages/cookies-tooltip/cookies-tooltip.tsx

```diff
@@ -44,17 +41,13 @@ export const CookiesTooltip: FC = () => {
   return (
     <Wrap>
       <Box>
-        <CookieIconWrap>
-          <ContentTheme
-            darkContent={<CookieInverse />}
-            lightContent={<Cookie />}
-          />
-        </CookieIconWrap>
         <Text>
-          We use cookies to collect anonymous site visitation data
-          to&nbsp;improve performance of&nbsp;our&nbsp;website. For&nbsp;more
-          info, read our&nbsp;
-          <Link href='https://lido.fi/privacy-notice'>Privacy Notice</Link>
+          Cookies are used to collect anonymous site visitation data
+          to&nbsp;improve performance of&nbsp;website. For&nbsp;more info,
```

Comment:
`of the website` or `website performance`

Reply:
Done